app/screens/receive-screen/receive-screen.tsx

```diff
@@ -133,7 +133,7 @@ export class ReceiveScreen extends React.Component<ReceiveScreenProps, {}> {
             />
             <Sheet style={SHEET}>
               <QRCode
-                value={likerID}
+                value={`{"likerId":"${likerID}","skipToConfirm":false}`}
```

Comment:
I think `skipToConfirm` can be left as default true if https://github.com/likecoin/likecoin-app/pull/136#pullrequestreview-372057157 was handled...

Reply:
Better use
``value={`{"likerId":${JSON.stringify(likerID)},"skipToConfirm":false}`}``
to prevent possible injection in the future

Reply:
@nnkken should use `JSON.stringify()` on entire object instead of just likerId